controllers/cluster_controller_phases_test.go

```diff
@@ -370,3 +370,186 @@ func TestClusterReconciler_reconcilePhase(t *testing.T) {
 		})
 	}
 }
+
+func TestClusterReconcilePhases_reconcileFailureDomains(t *testing.T) {
+	t.Run("reconcile infrastructure failure domain", func(t *testing.T) {
```

Comment:
Any particular reason to wrap the whole test under this `t.Run`?

Reply:
Ack removed